include/rocksdb/db.h

```diff
@@ -1741,8 +1741,16 @@ class DB {
       std::unique_ptr<Replayer>* /*replayer*/) {
     return Status::NotSupported("NewDefaultReplayer() is not implemented.");
   }
+#endif  // !ROCKSDB_LITE
 
-#endif  // ROCKSDB_LITE
+  // Experimental and subject to change
+  // Wait for the background flush and Compaction to finish.
```

Comment:
I don't understand. Does it wait until the currently running background jobs finish or until there are no running background jobs? And that might be an instant until the next background job is started, right?

If there is no running flush or compaction, does it "wait for flush and/or compaction" as its name implies?

Reply:
(I constantly find myself reverse-engineering the TEST functions also because they have completely unclear meaning.)

Reply:
> And that might be an instant until the next background job is started, right?

And it might never happen, right?